wasmedge-ggml-llama-interactive/src/main.rs

```diff
@@ -0,0 +1,55 @@
+use std::env;
+use std::io;
+use wasi_nn;
+
+fn read_input() -> String {
+    loop {
+        let mut answer = String::new();
+        io::stdin()
+            .read_line(&mut answer)
+            .ok()
+            .expect("Failed to read line");
+        if !answer.is_empty() && answer != "\n" && answer != "\r\n" {
+            return answer;
+        }
+    }
+}
+
+fn main() {
+    let args: Vec<String> = env::args().collect();
+    let model_name: &str = &args[1];
+
+    let graph =
+        wasi_nn::GraphBuilder::new(wasi_nn::GraphEncoding::Ggml, wasi_nn::ExecutionTarget::CPU)
```

Comment:
I think the execution target should be read from the command line? You already have to specify CPU in the pre-load setting in the CLI. 

But there is a bigger question: why do we ask the user to choose the target at all? If the underlying ggml library is compiled for GPU, this should be automatically executed on a GPU. Same goes for Mac, TPU, NPU etc.